examples/mpi/candle-nvidia/Dockerfile

```diff
@@ -0,0 +1,138 @@
+# ch-test-scope: full
+FROM nvcr.io/hpc/candle:20180326
+
+# The primary goals of this Dockerfile are:
+#   
+#    1. Provide users with a CANDLE image that builds from
+#       start to finish without user intervention.
+#
+#    2. Supply data before runtime. Enable the container to
+#       run the Pilot1 P1B1 and Pilot3 P3B1 benchmarks on systems 
+#       without network access at run time.
+#       
+# This Dockerfile is an extenstion of NVIDIA's CANDLE Dockerfile, 
+# which can be found on their GPU catalog at:
+#   
+#    - https://ngc.nvidia.com/catalog/containers/hpc%2Fcandle
+#
+# This Dockerfile requires an authenticated docker login to nvcr.io.
```

Comment:
We'd want to add instructions on how to get this, and skip the image if not logged in.